pkg/backup/testutils/br.go

```diff
@@ -51,6 +51,13 @@ func GenValidStorageProviders() []v1alpha1.StorageProvider {
 				SecretName: "gcs",
 			},
 		},
+		{
```

Comment:
What's the difference with that in L40-L44?

Reply:
Unnecessary test case, removed!